kornia/augmentation/__init__.py

```diff
@@ -3,6 +3,7 @@
     ColorJiggle,
     ColorJitter,
     Denormalize,
+    GaussianBlur,
```

Comment:
I’d say that we provide only RandomGaussianBlur — if the needed feature is sample batches of sigma then let’s upgrade the current functionality as you did. In Kornia world, to me plane GaussianBlur should be the Module version of Kornia.filters.gaussian_blur2d /cc @shijianjian @ducha-aiki